varnish/test_varnish.py

```diff
@@ -102,6 +109,14 @@ def test_check(self):
         for mname in COMMON_METRICS:
             self.assertMetric(mname, count=1, tags=['cluster:webs', 'varnish_name:default'])
 
+        config['instances'][0]['metrics_filter'] = ['SMA.*']
```

Comment:
Well good catch. It looks like contrary to what's written in the doc, the exclusion filter does not work with varnish >= 4 ...